restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/RecordingEndpointTest.java

```diff
@@ -128,6 +128,35 @@ public void getRecordingListFilteredByCallSid() {
         assertTrue(filteredRecordingsByCallSid.get("end").getAsInt() == 17);
     }
 
+    @Test
+    public void deleteRecording() {
+        JsonObject firstPage = RecordingEndpointTool.getInstance().getRecordingList(deploymentUrl.toString(), adminAccountSid,
+                adminAuthToken);
+        int totalSize = firstPage.get("total").getAsInt();
+        JsonArray firstPageRecordingsArray = firstPage.get("recordings").getAsJsonArray();
+        int firstPageRecordingsArraySize = firstPageRecordingsArray.size();
+        assertTrue(firstPageRecordingsArraySize == 34);
+        assertTrue(firstPage.get("start").getAsInt() == 0);
+        assertTrue(firstPage.get("end").getAsInt() == 34);
+        assertTrue(totalSize == 34);
+
+        //Recording SID to delete RE10000000000000000000000000000032
+
+        String recordingSid = "RE10000000000000000000000000000032";
+
+        RecordingEndpointTool.getInstance().deleteRecording(deploymentUrl.toString(), adminAccountSid, adminAuthToken, recordingSid);
+
+        firstPage = RecordingEndpointTool.getInstance().getRecordingList(deploymentUrl.toString(), adminAccountSid,
```

Comment:
i would just query for the recordign sid, and check that is 404,instead of basing the asserts on list